lib/kino/raw.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Kino.Raw do
```

Comment:
Should we call it `Kino.Inspect` then?

I think it would go better with the description.

Reply:
Also, should we make it so Kino.inspect uses this? :D

Reply:
`Kino.Inspect` sounds good! `Kino.inspect` handles optional label, so I'd keep it as is.